client/landing/stepper/hooks/use-prepare-site-for-migration.ts

```diff
@@ -44,9 +46,63 @@ export const usePrepareSiteForMigration = ( siteId?: number ) => {
 		enabled: Boolean( pluginInstallationState.completed ),
 	} );
 
+	const [ transferTimingTracked, setTransferTimingTracked ] = useState( false );
+	const [ siteTransferStart, setSiteTransferStart ] = useState( 0 );
+	const [ siteTransferEnd, setSiteTransferEnd ] = useState( 0 );
+	const [ pluginInstallationStart, setPluginInstallationStart ] = useState( 0 );
+	const [ pluginInstallationEnd, setPluginInstallationEnd ] = useState( 0 );
```

Comment:
We should use useRef when the state changes should not trigger new renders. 
It is way more light processing than useState.